app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class QuotesController < ApplicationController
+  def index
```

Comment:
See my comments in your facade. If you were to implement the passing of two things at that level, your controller might look something like:
```
 def index
   @search_phrase = params[:query]
   @total_found, @quotes = QuotesFacade.quote_search(@search_phrase)
end
```